packages/react/src/utils.ts

```diff
@@ -539,3 +539,24 @@ export type RelatedFieldColumn = {
 export const isRelatedFieldColumn = (value: any): value is RelatedFieldColumn => {
   return typeof value === "object" && value !== null && "field" in value && "relatedField" in value;
 };
+
+/**
+ * Humanizes a camelCase string by adding spaces between words and capitalizing the first letter
+ * Examples
+ *    humanizeCamelCase("createNewWidget") => "Create New Widget"
+ *    humanizeCamelCase("do99Things") => "Do 99 Things"
+ */
+export const humanizeCamelCase = (camelCaseString: string): string => {
```

Comment:
Given the number of regex operations in this function, it's quite difficult to understand the expected results.

Maybe add a test to prove that the results are expected?

Reply:
Added a jest test to prove these examples 👍